brave/ui/app/store/actions.js

```diff
@@ -47,6 +66,13 @@ function showModal (payload) {
     }
   }
 
+  // XXX for testing
+  if (payload.name === 'ACCOUNT_DETAILS') {
+    return {
+      type: MetaMaskActions.TOGGLE_PROVIDER,
+    }
+  }
```

Comment:
This needs to be removed before being merged. This hijacks the account details button to switch back and forth between the external provider wallet view.